portfolio/src/main/webapp/index.html

```diff
@@ -1,74 +1,105 @@
-        </li>
-        <li>
-          <h3>Roobeks Timer</h3>
-          <h4>Personal Project</h4>
-          <p>Easy to use timer and tool for Rubik's Cube solves. Tech Stack: HTML, CSS, JavaScript</p>
-        </li>
-      </ul>
-
-      <hr>
-
-
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Malcolm Jones Portfolio</h1>
+
+        <img src="/images/malcolm-1.jpeg" width="500" height="500">
+
+        <h2>About Me</h2>
+        <p>
+          Welcome to my portfolio website! I am Malcolm and I am from 
+          Winston-Salem, NC. Currently I am studying computer science at North 
+          Carolina Agricultural and Technical State Universty. 
+          Another passion of mine aside from computer science is the environment
+          and sustainability. My career aspiration is to use my skills and 
+          talents in computer science to solve problems related to the environment
+          and sustainability. Outside of my academics I enjoy going for walks, 
+          reading, and cooking. 
+        </p>
+
+        <h4>Connect with Me</h4>
+        <a href="https://www.linkedin.com/in/malcolmrjones" target="_blank">LinkedIn</a>
+        <a href="https://www.github.com/malcolmrjones" target="_blank">Github</a>
+        
+        <hr>
+
+        <h2>My Projects</h2>
+        <ul>
+          <li>cipes</h3>
+            <h4>REU - University of North Carolina at Charlotte</h4>
+            <p>
+              Web app that provides recipe recommendations based off of 
+              ingredient preferences. Tech Stack: HTML, CSS, JavaScript, 
+              Django Web Framework
+            </p>
+          </li>
+          <li>
+            <h3>Pomoro</h3>
+            <h4>Personal Project</h4>
+            <p>
+              Online timer for using the pomodoro technique. It focuses on 
+              simplicity and usability. Tech Stack: HTML, CSS, JavaScript, Vue.js
+            </p>
+          </li>
+          <li>
+            <h3>BBConnection</h3>
+            <h4>Hackathon - HackNCAT 2018</h4>
+            <p>
+              Web platform that connects small and medium black-owned 
+              businesses with opportunities. Tech Stack: HTML, CSS, JavaScript
+            </p>
+          </li>
+          <li>
+            <h3>Rolisto</h3>
+            <h4>Personal Project</h4>
+            <p>
+              Web app to create and manage task and to-do lists. 
+              Tech Stack: HTML, CSS, JavaScript, Angular, Firebase Firestore
+            </p>
+          </li>
+          <li>
+            <h3>Roobeks Timer</h3>
+            <h4>Personal Project</h4>
+            <p>
+              Easy to use timer and tool for Rubik's Cube solves. 
+              Tech Stack: HTML, CSS, JavaScript
+            </p>
+          </li>
+        </ul>
+
+        <hr>
+
```

Comment:
nit: remove extra new lines. We should always use only one new line to split loigc blocks.